source/get-started.txt

```diff
@@ -42,4 +34,408 @@ MongoDB Atlas and query data in your cluster.
+         find_package(mongoc-1.0 CONFIG REQUIRED)
+         target_link_libraries(my-application
+             PRIVATE $<IF:$<TARGET_EXISTS:mongo::mongoc_shared>,mongo::mongoc_shared,mongo::mongoc_static>)
+        
+      .. note::
+        
+         You can use the ``$<IF:$<TARGET_EXISTS...>:...>`` generator
+         expression in the preceding example to switch the link type of
+         ``libmongoc`` based on the form that is available from the
+         ``find_package()`` command. ``libmongoc`` supports building with
+         both *dynamic* and *static* library types, but vcpkg
+         installs only one type at a time.
+        
+      To configure the CMake project with vcpkg integration, set the CMake
+      toolchain file in the configuration command by running the following
+      commands in your shell:
+        
+      .. code-block:: bash
+
+         cmake -S . -B _build -D
+         CMAKE_TOOLCHAIN_FILE=$VCPKG_ROOT/scripts/buildsystems/vcpkg.cmake
+            
+   .. tab:: MacOS
+      :tabid: install-mac
+
+      You can download the {+driver-short+} on MacOS by using the `Homebrew
+      package manager <https://brew.sh/>`__. To do so, run the following command in
+      your shell:
+
+      .. code-block:: bash
+
+          brew install mongo-c-driver
+
+After you complete these steps, you have the {+driver-short+} installed 
+on your machine.
+
+.. _c-get-started-create-deployment:
+
+Create a MongoDB Deployment
+---------------------------
+
+You can create a free tier MongoDB deployment on MongoDB Atlas
+to store and manage your data. MongoDB Atlas hosts and manages
+your MongoDB database in the cloud.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a Free MongoDB deployment on Atlas
+
+      Complete the :atlas:`Get Started with Atlas </getting-started>`
+      guide to set up a new Atlas account and load sample data into a new free
+      tier MongoDB deployment.
+      
+   .. step:: Save your Credentials
+
+      After you create your database user, save that user's 
+      username and password to a safe location for use in an upcoming step.
+  
+After you complete these steps, you have a new free tier MongoDB
+deployment on Atlas, database user credentials, and sample data loaded
+into your database.
+
+
+.. _c-get-started-connection-string:
+
+Create a Connection String
+--------------------------
+
+You can connect to your MongoDB deployment by providing a
+**connection URI**, also called a *connection string*, which
+instructs the driver on how to connect to a MongoDB deployment
+and how to behave while connected.
+
+The connection string includes the hostname or IP address and 
+port of your deployment, the authentication mechanism, user credentials 
+when applicable, and connection options.
+
+To connect to an instance or deployment not hosted on Atlas, see the
+:ref:`c-connection-targets` guide.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Find your MongoDB Atlas Connection String
```

Comment:
```suggestion
   .. step:: Find Your MongoDB Atlas Connection String
```
S: I would standardize how the titles are capitalized. Some step titles are capitalized, others are not